test/evergreen.yml

```diff
@@ -2981,7 +2961,7 @@ buildvariants:
 - name: compatibility-tests
   display_name: Compatibility tests
   run_on:
-  - ubuntu1804-test
+  - ubuntu2004-test
```

Comment:
I thought there's a known issue of migrating compatibility tests?

Reply:
There is an issue with Compatibility tests (less frequent). Compatibility tests have no issues.

Reply:
Got it, thanks